Test/NUnit3.Tests.ps1

```diff
@@ -44,22 +46,28 @@ function Init
     $script:reportGeneratorVersion = $null
     $script:reportGeneratorArgument = $null
     $script:nunitVersion = $null
+    $script:supportNUnit2 = $false
 
     $script:buildRoot = $TestDrive.FullName
 
     $script:outputDirectory = Join-Path -Path $buildRoot -ChildPath '.output'
 
-    Copy-Item -Path (Join-Path -Path $PSScriptRoot -ChildPath 'Assemblies\NUnit3*\bin\*\*') -Destination $buildRoot
+    # Test assemblies in separate folders to avoid cross-reference of NUnit Framework assembly versions
+    @(3, 2) | ForEach-Object  {
+        New-Item (Join-Path $buildRoot "NUnit$($_)Tests") -Type Directory

```

Comment:
Do we need to add these to `.gitignore`?

Reply:
Still copied to TestDrive - should not end up in repo path anywhere.